server/channels/app/shared_channel.go

```diff
@@ -16,12 +16,16 @@ import (
 	"github.com/mattermost/mattermost/server/v8/channels/store"
 )
 
-func (a *App) getSharedChannelsService() (SharedChannelServiceIFace, error) {
+func (a *App) getSharedChannelsService(ensureIsActive bool) (SharedChannelServiceIFace, error) {
```

Comment:
Has this been added in anticipation it will be needed for a future PR?  All calls currently are `false`.

Reply:
@wiggin77 the option was left just for flexibility, but I don't have a follow up PR in mind that would use this, happy to remove it